src/Faithlife.WebRequests/CookieUtility.cs

```diff
@@ -17,18 +18,18 @@ public static class CookieUtility
 		/// <returns>A string in the Set-Cookie HTTP header format.</returns>
 		public static string ToSetCookieHeaderValue(this Cookie cookie)
 		{
-			string strHeader = cookie.ToString();
+			StringBuilder strHeader = new StringBuilder(cookie.ToString());
 			if (!string.IsNullOrEmpty(cookie.Domain))
-				strHeader += "; domain=" + cookie.Domain;
+				strHeader.Append($"; domain={cookie.Domain}");
```

Comment:
If the goal is to reduce string allocations, we should chain `Append`s instead of using string interpolation.
```suggestion
				strHeader.Append("; domain=").Append(cookie.Domain);
```